lerna.json

```diff
@@ -4,6 +4,9 @@
       "skip-npm": true,
       "conventionalCommits": true,
       "message": "chore(release): version %v"
+    },
+    "version": {
+      "changelog-preset": "angular"
```

Comment:
> By default, the changelog preset is set to angular. In some cases you might want to change either use a another preset or a custom one.

source : https://github.com/lerna/lerna/tree/master/commands/version#--changelog-preset

Reply:
The generated logs proves that its not the case :(
I guess that the documentation is on lerna is not up to date :recycle:

Reply:
alrite !